recipe/meta.yaml

```diff
@@ -31,18 +31,18 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - protobuf
-    - libprotobuf
-    - pytest-runner
-    - ninja
-    - pybind11
-    - numpy
+    - protobuf 3.20.3
+    - libprotobuf 3.20.3
```

Comment:
This shouldn't be pinned in that case, because we want to use the pinning specified in conda_build_config.yaml.